OSRMTextInstructions.podspec

```diff
@@ -0,0 +1,47 @@
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.name = "OSRMTextInstructions.swift"
+  s.version = "0.0.1"
+  s.summary = "Project OSRM text instructions"
+
+  s.description  = <<-DESC
+  OSRM Text Instructions is a library for iOS, macOS, tvOS, and watchOS applications written in Swift or Objective-C that transforms OSRM route responses into localized text instructions.
+                DESC
+
+  s.homepage = "http://project-osrm.org/"
+
+  # ―――  Spec License  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.license = { :type => "ISC", :file => "LICENSE.md" }
+
+  # ――― Author Metadata  ――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.author = { "Mapbox" => "mobile@mapbox.com" }
```

Comment:
If we’re going to have Mapbox as the `author`, let’s include [a corresponding `social_media_url`](https://github.com/mapbox/MapboxDirections.swift/blob/0f72b8d0c6c51b4f735ec441cd04bbb58e1ca966/MapboxDirections.swift.podspec#L22).